ts/components/search/SearchErrorMessage.tsx

```diff
@@ -0,0 +1,73 @@
+import { Text, View } from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import I18n from "../../i18n";
+import customVariables from "../../theme/variables";
+
+const styles = StyleSheet.create({
+  contentWrapper: {
+    flex: 1,
+    alignItems: "center"
+  },
+  message: {
+    fontSize: customVariables.fontSizeBase,
+    padding: customVariables.contentPadding,
+    textAlign: "center"
+  }
+});
+
+export enum ErrorSearchType {
+  InvalidSearchBarText,
+  NoResultsFound
+}
+
+/**
+ * Print the icon according to current error type
+ * @param errorType
+ */
+export const renderIconErrorSearch = (errorType: ErrorSearchType) => {
```

Comment:
As little improvement you convert `ErrorSearchType` from enum to union type, ex:
```javascript
type ErrorType = "InvalidSearchBarText" | "NoResultsFound";
```
Using enum `renderIconErrorSearch` should be unsafe: for example you can do `renderIconErrorSearch(100)`. This because enums are converted into numbers a runtime
In the other hand you can't call `renderIconErrorSearch` with a parameter different from one specified into the union

Reply:
Done. In the commit f3a51b5